packages/block-editor/src/components/block-tools/index.js

```diff
@@ -95,6 +95,17 @@ export default function BlockTools( {
 		} else if (
 			isMatch( 'core/block-editor/delete-multi-selection', event )
 		) {
+			/**
+			 * Check if the target element is a text area, input or
+			 * event.defaultPrevented and return early. In all these
+			 * cases backspace could be handled elsewhere.
+			 */
+			if (
+				[ 'INPUT', 'TEXTAREA' ].includes( event.target.nodeName ) ||
```

Comment:
Should we add a check for content editable? Less common I guess.
Should we add an e2e test?

Reply:
Yeah I think it's quite uncommon. If the `content editable` thing is raised let's add it then.

Regarding the tests you're 100% right and I should have written them in this PR, but [this follow up](https://github.com/WordPress/gutenberg/pull/37624) will do I guess 😄